docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -1,1107 +0,0 @@
-{
```

Comment:
I think there are files in aux_files subfolder of tutorials that may be obsolete once the tutorial is deleted. Eg lattice_protein.png that we should probably clean up too.